models/runs/grouping.mdx

```diff
@@ -1,133 +1,146 @@
+                    "acc": 0.5 + (step / 100) * 0.3 + (i * 0.05),
+                    "loss": 1.0 - (step / 100) * 0.5
+                })
+```
 
-### 4. Manage groups in the UI
+Within the project's workspace, you can view runs organized by group. The following image shows the runs table grouped by the group name. Three groups named `A`, `B`, and `C` appear in the runs table, each containing three runs.
 
-#### Manage a run
-After a run is initialized, you can move it to a new group from your workspace or its **Runs** page.
+<Frame>
+    <img src="/images/runs/group_name_3_groups.png" alt="Runs table grouped by group name"  />
+</Frame>
+</Tab>
+<Tab title="W&B App">
 
 1. Navigate to your W&B project.
-1. Select the **Workspace** or **Runs** tab from the project sidebar.
-1. To move a single run, search or scroll to it. Hover over the run name, click the three vertical dots, then click **Move to another group**.
-1. To create a new group, click **New group**. Type a group name, then submit the form.
-1. To delete a run, click the three vertical dots, then click **Delete run**.
-1. To move a run, select the new group from the list, then click **Move**.
-
-#### Manage runs in bulk
-This section shows how to manage multiple runs at once, across one or more run groups.
-
-1. If necessary, [toggle grouping by columns](#toggle-grouping-by-column-in-the-ui), grouping by the **Group** column.
-1. Expand a group to view its runs. In a single bulk operation, you can move runs from different groups to the same target group. Expand all relevant groups.
-1. To select all runs in a group, click the checkbox next to the group name.
-1. To select individual runs, click their checkboxes.
-1. To select all runs in all groups, click the checkbox next to the **Name** column heading.
-1. Above the table, choose a bulk operation: **Tag**, **Move to group**, **Move to project**, or **New sweep**.
-
-#### Create a group
-To create a new run group:
+2. Select the **Runs** tab from the project sidebar.
+3. Above the list of runs, click the **Group** button.
+4. Click the checkbox next to one or more runs you want to group.
+5. Select **Move to group**.
+6. In the modal, select an existing group or create a new group.
+</Tab>
+</Tabs>
+
+
+### View groups
+
+View runs organized by group in the W&B App:
+
+1. In your project sidebar, select the **Runs** tab.
+2. Above the list of runs, click the **Group** button.
+3. From the dropdown, select either **Group**.
+
+### Move runs between groups
+
+Move runs in one group to another group:
+
 1. Navigate to your W&B project.
-1. Select the **Workspace** or **Runs** tab from the project sidebar.
-1. Click **New group**. Type a group name, then submit the form.
+1. Select the **Runs** tab from the project sidebar.
+1. Select one or more runs by clicking their checkboxes.
+1. Above the table, click **Move to group**.
+1. Within the modal, select the target group or create a new group.
+1. Click **Move**.
+
+### Remove a group
```

Comment:
This title could use some disambiguation from the one on line 91. This is something like "Remove runs from a group" or "Stop grouping by a group"?